packages/SwingSet/src/vats/vat-admin/vat-vat-admin.js

```diff
@@ -42,6 +42,7 @@ export function buildRootObject(vatPowers, _vatParameters, baggage) {
   function noteRunningVat(vatID) {
     const prr = producePRR();
     const [doneP] = prr;
+    // @ts-expect-error may not have `catch`
```

Comment:
Please type `producePRR` to return a tuple instead of an array, e.g. with `/** @type {const} */` on the return value.